### PR TITLE
doc: crypto: Add missing ocrypto APIs to doxygen

### PR DESCRIPTION
**Pull request by mswarowsky**
Several API docs were missing for the ocrypto library added them

Ref: NCSDK-21222

### CODE REVIEW

**Review by SebastianBoe — APPROVED** [review]

File: crypto/doc/api.rst [hunk 1]
```diff
@@ -115,6 +115,20 @@ AES - Advanced Encryption Standard APIs
    :project: nrfxlib
    :members:
 
+AES-CBC - AES Cipher Block Chaining Mode
```
Comment (mia-ko) [inline crypto/doc/api.rst:118]: Most of the sections in the file have APIs in the end, maybe we could add it to all new ones as well for consistency? Except perhaps the ocrypto internal types...

Comment (mia-ko) [reply crypto/doc/api.rst:118]: Another question is if it should be API and not APIs when only one doxygengroup is presented in the section. But this is relevant for the entire file and does not have to be addressed in this PR.

**Review by mia-ko — CHANGES_REQUESTED** [review]

File: crypto/doc/api.rst (hunk 1 above)
Comment (mswarowsky) [reply crypto/doc/api.rst:118]: Then this is in the API section anyway so maybe it would be better to just remove APIs/API as it should be obvious

**Review by mia-ko — APPROVED** [review]

File: crypto/doc/api.rst (hunk 1 above)
Comment (mia-ko) [reply crypto/doc/api.rst:118]: True

Comment (mswarowsky) [reply crypto/doc/api.rst:118]: Ok removed all the API(s) in the titles

**Pull request merged**